### PR TITLE
Copyright update

### PR DESCRIPTION
**Pull request by jozefbakus**
Copyright update

### CODE REVIEW

**Review by kevinwallimann — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
+        Copyright 2018 ABSA Group Limited
```
Comment (Zejnilovic) [inline README.md]: Why put README under copyright?

File: pom.xml [hunk 2]
```diff
@@ -1,20 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
+  ~ Copyright 2018 ABSA Group Limited
```
Comment (Zejnilovic) [inline pom.xml:3]: Shouldn't copyright be the topmost things, above the xml def?

Comment (jozefbakus) [reply pom.xml:3]: It was automatically added there.
If I move it to topmost position it says "Xml declaration should precede all document content"

File: README.md (hunk 1 above)
Comment (jozefbakus) [reply README.md]: Removed!

**Review by Zejnilovic — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (Zejnilovic) [reply pom.xml:3]: Let's believe the IDE

**Pull request merged**